linux/LinuxProcess.h

```diff
@@ -18,17 +18,18 @@ in the source distribution for its full text.
 #include "linux/IOPriority.h"
 
 
-#define PROCESS_FLAG_LINUX_IOPRIO    0x00000100
-#define PROCESS_FLAG_LINUX_OPENVZ    0x00000200
-#define PROCESS_FLAG_LINUX_VSERVER   0x00000400
-#define PROCESS_FLAG_LINUX_CGROUP    0x00000800
-#define PROCESS_FLAG_LINUX_OOM       0x00001000
-#define PROCESS_FLAG_LINUX_SMAPS     0x00002000
-#define PROCESS_FLAG_LINUX_CTXT      0x00004000
-#define PROCESS_FLAG_LINUX_SECATTR   0x00008000
-#define PROCESS_FLAG_LINUX_LRS_FIX   0x00010000
-#define PROCESS_FLAG_LINUX_DELAYACCT 0x00040000
-#define PROCESS_FLAG_LINUX_AUTOGROUP 0x00080000
+#define PROCESS_FLAG_LINUX_IOPRIO      0x00000100
+#define PROCESS_FLAG_LINUX_OPENVZ      0x00000200
+#define PROCESS_FLAG_LINUX_VSERVER     0x00000400
+#define PROCESS_FLAG_LINUX_CGROUP      0x00000800
+#define PROCESS_FLAG_LINUX_OOM         0x00001000
+#define PROCESS_FLAG_LINUX_SMAPS       0x00002000
+#define PROCESS_FLAG_LINUX_CTXT        0x00004000
+#define PROCESS_FLAG_LINUX_SECATTR     0x00008000
+#define PROCESS_FLAG_LINUX_LRS_FIX     0x00010000
+#define PROCESS_FLAG_LINUX_DELAYACCT   0x00040000
+#define PROCESS_FLAG_LINUX_AUTOGROUP   0x00080000
+#define PROCESS_FLAG_LINUX_ISCONTAINER 0x00100000
```

Comment:
I'd prefer if we could avoid this alignment change by keeping the new name short enough. `PROCESS_FLAG_LINUX_CONTAINER` would fit the bill IMHO.